azure/template.json

```diff
@@ -158,7 +174,7 @@
             }
         },
         {
-            "apiVersion": "2017-05-10",
```

Comment:
You'll need a resourceGroup key for this resource and the other missing resources